--- newrelic-mysql-plugin.debian	2013-10-30 12:30:33.000000000 -0700
+++ newrelic-mysql-plugin.tmp	2013-10-30 12:40:20.000000000 -0700
@@ -25,6 +25,9 @@
 JAVA=/usr/bin/java
 DAEMONDIR=/path/to/plugins
 
+# Read configuration variable file if it is present
+[ -r /etc/default/$NAME ] && . /etc/default/$NAME
+
 # Exit if the package is not installed
 if test ! -x $JAVA -o ! -e $DAEMONDIR/$DAEMON 
 then
@@ -32,9 +35,6 @@
     exit 0
 fi
 
-# Read configuration variable file if it is present
-[ -r /etc/default/$NAME ] && . /etc/default/$NAME
-
 . /lib/init/vars.sh
 . /lib/lsb/init-functions
 
